www/js/controllers.js

```diff
@@ -24,22 +24,35 @@ angular.module('wpApp.controllers', [])
 
   // Sites view: templates/sites.html
 
-  $scope.sites = [];
-
-  // Create a cache for our site information
-  if (!CacheFactory.get('siteCache')) {
-    CacheFactory.createCache('siteCache');
-  }
-
-  $rootScope.siteCache = CacheFactory.get('siteCache');
-
-  if( $rootScope.siteCache ) {
-    // Loop through storage and add to our sites
-    angular.forEach( $rootScope.siteCache.keys(), function( value, key ) {
-      $scope.sites.push( $rootScope.siteCache.get(value) );
-    });
-  }
-
+  
+  $scope.loadSites = function() {
```

Comment:
What's the purpose of binding this function to the scope? Are you gonna use it in a view somewhere?